Rules/UseConsistentIndentation.cs

```diff
@@ -220,16 +220,22 @@ public override IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string file
                 // Check if the current token matches the end of a PipelineAst
                 var matchingPipeLineAstEnd = pipelineAsts.FirstOrDefault(pipelineAst =>
                         PositionIsEqual(pipelineAst.Extent.EndScriptPosition, token.Extent.EndScriptPosition)) as PipelineAst;
-                if (matchingPipeLineAstEnd != null)
+                if (matchingPipeLineAstEnd == null)
                 {
-                    if (pipelineIndentationStyle == PipelineIndentationStyle.IncreaseIndentationForFirstPipeline)
-                    {
-                        indentationLevel = ClipNegative(indentationLevel - 1);
-                    }
-                    else if (pipelineIndentationStyle == PipelineIndentationStyle.IncreaseIndentationAfterEveryPipeline)
-                    {
-                        indentationLevel = ClipNegative(indentationLevel - (matchingPipeLineAstEnd.PipelineElements.Count - 1));
-                    }
+                    break;
+                }
+                var pipelineSpansOnlyOneLine = matchingPipeLineAstEnd.Extent.StartLineNumber == matchingPipeLineAstEnd.Extent.EndLineNumber;
```

Comment:
```suggestion
                bool pipelineSpansOnlyOneLine = matchingPipeLineAstEnd.Extent.StartLineNumber == matchingPipeLineAstEnd.Extent.EndLineNumber;
```

Reply:
If this is used only once, why not keep it inside the `if` expression? `if (...)` is shorter than `var myVariableName = ...;`